### ISSUE TITLE
how to use crypto??

### ISSUE THREAD

**Issue by D33M0N** [pcloudcom/console-client#136]
I fail to find an option or guide or --help command, how to turn crypto on for some specific folder. I don't want to crypt the entire drive. How does this work?

**Comment by Lenezir** [comment]
Hey,
After you started it with the -p and -s options to record your password the very first time, like `pcloudcc -u example@myemail.com -p -s`, you start it again with the -c option `pcloudcc -u example@myemail.com -c`.
It will ask you the password of your Crypto, the encrypted folder will mount on your pCloud drive.
Cheers.

**Comment by D33M0N** [comment]
yes yes... but ... I don't want to crypt everything obviously. so how can I define what to crypt and what not?

**Comment by Lenezir** [comment]
Put the files you want to crypt into the Crypto folder, and let the rest outside, at the root.
If you want to crypt something without upsetting your tree structure, you have to use an external tool.
